script.js

```diff
@@ -0,0 +1,67 @@
+// Retrieve books from localStorage or initialize an empty array
+const books = JSON.parse(localStorage.getItem('books')) || [];
+
+// Function to display the books in the collection
+function displayBooks() {
+  const booksDiv = document.getElementById('books');
+  booksDiv.innerHTML = ''; // Clear the booksDiv before displaying the books
+
+  if (books.length === 0) {
+    booksDiv.textContent = 'No books in the collection.';
+    return;
+  }
+
+  // Function to remove a book from the collection
+  const removeBook = (index) => {
+    books.splice(index, 1);
+    localStorage.setItem('books', JSON.stringify(books));
+    displayBooks();
+  };
+
+  books.forEach((book, index) => {
+    booksDiv.innerHTML += `<h2>All Books</h2>
```

Comment:
[OPTIONAL]
- [ ] Nice work displaying the books everything works well :clap: :sunglasses: However it would be nice to remove the `<h2>All Books</h2>`  from your JavaScript and add it to your html file. That will prevent repeating the "All Books" whenever adding new book.
### Your design :point_down: 
![Screenshot from 2023-06-05 12-40-38](https://github.com/kit0-0/awesomebook/assets/91142432/ed06527d-1785-4c9f-819d-3a6756d31bcf)
### It will be better if it looks like this instead :point_down: 
![Screenshot from 2023-06-05 12-50-16](https://github.com/kit0-0/awesomebook/assets/91142432/15170eff-8d77-4f2e-afc1-eeb56a3c0eba)